Common/sources/license.js

```diff
@@ -32,28 +32,18 @@
 
 'use strict';
 
-const config = require('config');
-const configL = config.get('license');

```

Comment:
sketchy file changes :joy: (this appears to be from the tag, and not you)

Reply:
This is from the tag